docs/references/http_api_reference.md

```diff
@@ -1240,6 +1255,29 @@ curl --request GET \
   Unix timestamp for filtering documents created after this time. 0 means no filter. Defaults to `0`.
 - `create_time_to`: (*Filter parameter*), `integer`
   Unix timestamp for filtering documents created before this time. 0 means no filter. Defaults to `0`.
+- `suffix`: (*Filter parameter*), `array[string]`
+  Filter by file suffixes. Multiple values supported. Examples: `["pdf", "txt", "docx"]`. Defaults to all types.
+- `run`: (*Filter parameter*), `array[string]`
+  Filter by document processing status. Supports both numeric and text formats:
+  - Numeric format: `["0", "1", "2", "3", "4"]`
```

Comment:
- Numeric format: `[0, 1, 2, 3, 4]`

I think “” should be removed. Same for the rest.